python3/bin/perfmon

```diff
@@ -357,8 +360,10 @@ class RRDUpdates:
     def __repr__(self):
         return "<RRDUpdates object: params=%s>" % str(self.params)
 
-    def refresh(self, session, override_params={}):
+    def refresh(self, session, override_params=None):
```

Comment:
May I know how does pylint complain `override_params={}` ?
The current change makes the type of `override_params` ambiguous.

Or, would it be better to do the following:
```
params = {}
if override_parameter is not None:
    params = override_parameter
```

Reply:
This is a notorious problem with using mutable default values.
Briefly speaking, using mutable default values will cause problems where each call of the method will change the mutable value and will impact the other calls of the method.
See https://www.geeksforgeeks.org/use-mutable-default-value-as-an-argument-in-python/